src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,905 @@
+
+>>[[“What’s the deal with Toe? He not talking today?”|The Gutter - Bar - Darryl game]]
+>>[[“What do you dream about, Darryl?"|The Gutter - Bar - Darryl dream]]
+>>[[“Is Toe his real name?”|The Gutter - Bar - Darryl toe]]
+>>[[“Why don’t you like Ron?”|The Gutter - Bar - Darryl ron]]
+>>[["See ya, Darryl."|The Gutter - Bar hub]]
+
+::The Gutter - Bar - Darryl game
+Darryl shook his head. “Eh, he never says much. About the only thing he ever does is pour and play games.”
+
+“What kind of games?” I asked.
+
+“Oh, anything. He had a collection, but some little shits recently stole most of it. It’s gotten to the point where he’s only got the stuff for pool and he won’t give it up to anybody.”
+
+[[I took note of Darryl’s observations|The Gutter - Bar - Darryl]].
+
+::The Gutter - Bar - Darryl dream
+Darryl’s eyes widened and he gulped. He let his vision linger on the edge of the bar for a moment before he spoke.
+
+“Mostly Grimace, the dark purple beast spoken of in the Book of Revelations that you would get with every McDonalds happy meal. When Grimace comes on his dark chariot driven by McNuggets, those of us who haven’t supped of the fry and the flesh weekly will be punished. It will not be swift. It will be the opposite of mercy.”
+
+Darryl cleared his throat, as if he’d just spoken several sentences by accident.
+
+“Oh, and last night I had a sexy dream about the mom from Family Guy. Don’t tell nobody.”
+
+[[I made a mental note to never ask Darryl a personal question again|The Gutter - Bar - Darryl]].
+
+::The Gutter - Bar - Darryl toe
+“Well...” Darryl paused for dramatic effect, or maybe he forgot the story.
+
+“They call him Toe because he has none. He cut them off and tried to eat them during a nasty trip, the story goes.
+
+“I’ve heard he used to love to have chicks suck his toes and now he pays to have them suck the air where they used to be.”
+
+Toe loomed over us both and we shrank. He nodded gravely and cleared Darryl’s empty glass.
+
+As he was turning away he let out a low rumble. “It’s all true.”
+
+[[I had solved another mystery. Good practice, I guess|The Gutter - Bar - Darryl]]?
+
+::The Gutter - Bar - Darryl ron
+“That’s between me, Ron, and the packet of hot sauce he tried to shove down my trousers as a prank.” Darryl said plainly.
+
+“Oh, dangit.” Darryl shook his head. “I goofed.”
+
+[[I laughed very loud at Darryl’s goof|The Gutter - Bar - Darryl]].
+
+::The Gutter - Ron
+Sitting down next to Ron at the bar seemed weird. I just stood behind him instead, stuck for a moment on what to say.
+
+>>[["Hey. Fuck you, Ron."|The Gutter - Ron - Fuck you]]
+>>[["Uh, Ron, can I talk to you out back?"|The Gutter - Ron - Request to talk]]
+>>[[I pretended to shake Ron’s hand, then pulled back at the last second to fix my hair. “Too slow, fucko,” I said as I walked away smiling. It never gets old|The Gutter - Inside hub]].
```

Comment:
Yeah, that's a great idea! I think that sells the joke a lot better than just laying it all out. I knew something looked weird about this big block of joke that had no other interactive options. 

It's even easier to code actually, because the player can only see this scene once before they default to the "asking for a pool game" scene.

Reply:
Yeah, I'm with Nathan on this. Also, I noticed that this option appears as a visited link even before you try it because it takes you back to the main Gutter hub passage. 

On that note: how do y'all feel about making sure unvisited and visited links have the exact same styles? It's rare that we let players loop through the same decision anyway, and that would be an easy way to prevent telegraphing options (like this one) that unexpectedly just lead players back to where they've already been.

Reply:
I think styling all links to appear the same all the time is fine/good, ya.